Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+FROM gcr.io/distroless/static-debian12:nonroot
+
+# TARGETOS and TARGETARCH are set automatically when --platform is provided.
+ARG TARGETOS
+ARG TARGETARCH
+ARG PRODUCT_VERSION
+
+LABEL name="http-echo" \
+      maintainer="HashiCorp Consul Team <consul@hashicorp.com>" \
+      vendor="HashiCorp" \
+      version=$PRODUCT_VERSION \
+      release=$PRODUCT_VERSION \
+      summary="A test webserver that echos a response. You know, for kids." 
+
```

Comment:
Should we expose a default port and the "ECHO_TEXT" environment variable?

Reply:
Yep, I like it.

Reply:
I'm going to add a default of `hello-world`, which shouldn't break anyone passing these args already, but would make it easier for quick and dirty testing.